README.md

```diff
@@ -0,0 +1,63 @@
+# GA4GH SDK
+
+## Synopsis
+
+A Generic SDK and CLI for GA4GH API services
+
+## Basic usage
+
+## Installation
+
+## Building
+
+First, clone the repository, and then run the following command to automatically generate models using OpenAPI specifications:
+```
+bash ./build-models.sh
+```
+
+To build the project:
+```
+cargo build
+```
+
+## Running the tests
```

Comment:
I think for consistency with other section names, consider renaming this section to "Testing"